docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
@msecret - Just a quick follow up - I'm a curious soul - sorry for blowing up your inbox. Wanted to show some screenshots to isolate *when* it happens.

Before `$ npm install`

<img width="820" alt="screen shot 2016-05-09 at 5 03 29 pm" src="https://cloud.githubusercontent.com/assets/15252830/15128192/5bebe0c2-1608-11e6-8f09-0829b7ce4084.png">

After `$ npm install`

<img width="818" alt="screen shot 2016-05-09 at 5 05 04 pm" src="https://cloud.githubusercontent.com/assets/15252830/15128202/653eab82-1608-11e6-89fd-928978817c42.png">

The point at which the `/src/stylesheets/lib` directory appeared.

<img width="619" alt="screen shot 2016-05-09 at 5 05 18 pm" src="https://cloud.githubusercontent.com/assets/15252830/15128300/feea1f0a-1608-11e6-8711-07f704b08200.png">

Looks like `$ npm update` does not run the `$ gulp copy-vendor-sass` - from the npm `prestart` (??), which makes sense. To test that hypothesis - I did the following.

1. `$ bundle update` - to avoid the kramdown error (must be done prior to `$ npm start`.
1. `$ npm update`
1. `$ npm start`

It worked. Which makes sense as calling `$ gulp website:serve` does not appear to call `$ gulp copy-vendor-sass`. If the goal is to *not* call `gulp` directly - then it works as expected.